Utils/interface/ParticleMapper.h

```diff
@@ -21,27 +21,26 @@ namespace mithep {
   class ParticleMapper {
   public:
 
-    ParticleMapper();
+    ParticleMapper(Double_t DeltaEta = 0.3,Double_t DeltaPhi = 0.3,Double_t EtaMax = 5.0);
```

Comment:
Initialization of eta-phi bins does not have to take place simultaneously as the initialization of particle index containers. A better implementation would be to define a mapper for a specific binning, then to reuse it with different PFCandidate collections.